src/main/java/net/sf/jabref/exporter/IExportFormat.java

```diff
@@ -57,8 +58,7 @@
      *            should be exported. If null, all entries will be exported.
      * @throws Exception
      */
-    void performExport(BibDatabase database, MetaData metaData,
- String file, Charset encoding,
-            Set<String> entryIds) throws Exception;
+    void performExport(BibDatabase database, MetaData metaData, String file, Charset encoding, Set<String> entryIds,
+            JournalAbbreviationRepository repository) throws Exception;
```

Comment:
As neither Exporter nor Layouter are part of the logic package yet, I would not pass the loader around there.